README.md

```diff
@@ -1,2 +1,9 @@
 # dice-server
-The Code being used by the TripleA dice server
+
+The MARTI dice server.
+
+## Getting Started
+
+* [Setup and deployment](https://github.com/triplea-game/dice-server/tree/master/bin)
+* [Web server configuration](https://github.com/triplea-game/dice-server/tree/master/config/nginx)
```

Comment:
Can you add some details on a next iteration how to deploy the configuration? Which folder should it land in?

Reply:
Complete instructions for updating the configuration are at the target link.

Reply:
very nice, thanks